snooty/parser.py

```diff
@@ -347,6 +347,29 @@ def handle_directive(
                 msg = f'Invalid "literalinclude": {err}'
                 self.diagnostics.append(Diagnostic.error(msg, util.get_line(node)))
             return
+        elif name == "include":
+            if argument_text is None:
```

Comment:
Ideally this would be validated by the parser. The fact that we don't currently support that is a source of great personal shame